host/xtest/pkcs11_1000.c

```diff
@@ -3741,3 +3742,863 @@ static void xtest_pkcs11_test_1016(ADBG_Case_t *c)
+	},
+	{
+		.test_name = "CKM_MD5/test pattern",
+		.mecha = CKM_MD5,
+		.data = digest_test_pattern,
+		.data_size = sizeof(digest_test_pattern) - 1,
+		.digest = digest_test_pattern_md5,
+		.digest_size = sizeof(digest_test_pattern_md5),
+	},
+	{
+		.test_name = "CKM_SHA_1/empty",
+		.mecha = CKM_SHA_1,
+		.data = digest_test_pattern_empty,
+		.data_size = sizeof(digest_test_pattern_empty) - 1,
+		.digest = digest_test_pattern_empty_sha1,
+		.digest_size = sizeof(digest_test_pattern_empty_sha1),
+	},
+	{
+		.test_name = "CKM_SHA_1/test pattern",
+		.mecha = CKM_SHA_1,
+		.data = digest_test_pattern,
+		.data_size = sizeof(digest_test_pattern) - 1,
+		.digest = digest_test_pattern_sha1,
+		.digest_size = sizeof(digest_test_pattern_sha1),
+	},
+	{
+		.test_name = "CKM_SHA224/empty",
+		.mecha = CKM_SHA224,
+		.data = digest_test_pattern_empty,
+		.data_size = sizeof(digest_test_pattern_empty) - 1,
+		.digest = digest_test_pattern_empty_sha224,
+		.digest_size = sizeof(digest_test_pattern_empty_sha224),
+	},
+	{
+		.test_name = "CKM_SHA224/test pattern",
+		.mecha = CKM_SHA224,
+		.data = digest_test_pattern,
+		.data_size = sizeof(digest_test_pattern) - 1,
+		.digest = digest_test_pattern_sha224,
+		.digest_size = sizeof(digest_test_pattern_sha224),
+	},
+	{
+		.test_name = "CKM_SHA256/empty",
+		.mecha = CKM_SHA256,
+		.data = digest_test_pattern_empty,
+		.data_size = sizeof(digest_test_pattern_empty) - 1,
+		.digest = digest_test_pattern_empty_sha256,
+		.digest_size = sizeof(digest_test_pattern_empty_sha256),
+	},
+	{
+		.test_name = "CKM_SHA256/test pattern",
+		.mecha = CKM_SHA256,
+		.data = digest_test_pattern,
+		.data_size = sizeof(digest_test_pattern) - 1,
+		.digest = digest_test_pattern_sha256,
+		.digest_size = sizeof(digest_test_pattern_sha256),
+	},
+	{
+		.test_name = "CKM_SHA384/empty",
+		.mecha = CKM_SHA384,
+		.data = digest_test_pattern_empty,
+		.data_size = sizeof(digest_test_pattern_empty) - 1,
+		.digest = digest_test_pattern_empty_sha384,
+		.digest_size = sizeof(digest_test_pattern_empty_sha384),
+	},
+	{
+		.test_name = "CKM_SHA384/test pattern",
+		.mecha = CKM_SHA384,
+		.data = digest_test_pattern,
+		.data_size = sizeof(digest_test_pattern) - 1,
+		.digest = digest_test_pattern_sha384,
+		.digest_size = sizeof(digest_test_pattern_sha384),
+	},
+	{
+		.test_name = "CKM_SHA512/empty",
+		.mecha = CKM_SHA512,
+		.data = digest_test_pattern_empty,
+		.data_size = sizeof(digest_test_pattern_empty) - 1,
+		.digest = digest_test_pattern_empty_sha512,
+		.digest_size = sizeof(digest_test_pattern_empty_sha512),
+	},
+	{
+		.test_name = "CKM_SHA512/test pattern",
+		.mecha = CKM_SHA512,
+		.data = digest_test_pattern,
+		.data_size = sizeof(digest_test_pattern) - 1,
+		.digest = digest_test_pattern_sha512,
+		.digest_size = sizeof(digest_test_pattern_sha512),
+	},
+};
+
+static CK_ATTRIBUTE digest_generate_aes_object[] = {
+	{ CKA_CLASS, &(CK_OBJECT_CLASS) { CKO_SECRET_KEY },
```

Comment:
no space char between cast and cast'ed:
`	{ CKA_CLASS, &(CK_OBJECT_CLASS){ CKO_SECRET_KEY },`

Reply:
OK

Reply:
Just a note -- I have trigged my local copy to have more checkpatch stuff... and it complains
```
ERROR: space required before the open brace '{'
#24: FILE: host/xtest/pkcs11_1000.c:3934:
+	{ CKA_CLASS, &(CK_OBJECT_CLASS){ CKO_SECRET_KEY },
```
Did the change nevertheless.